### PR TITLE
Add pt-br/docs/reference/glossary/kubectl.md

### PR DESCRIPTION
**Pull request by MrErlison**
Add 
- pt-br/docs/reference/glossary/kubectl.md

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/kubectl.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+short_description: >
+  Uma ferramenta de linha de comando para se comunicar com um cluster Kubernetes.
+
+aka:
+- kubectl
+tags:
+- tool
+- fundamental
+---
+Ferramenta de linha de comando para se comunicar com o {{< glossary_tooltip text="plano de controle" term_id="control-plane" >}} de um cluster Kubernetes usando a API do Kubernetes.
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/kubectl.md:15]: "camada de gerenciamento" would be our agreed upon term in Portuguese for control plane.

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Minor change on the control plane translation.

File: content/pt-br/docs/reference/glossary/kubectl.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ tags:
 - tool
 - fundamental
 ---
-Ferramenta de linha de comando para se comunicar com o {{< glossary_tooltip text="plano de controle" term_id="control-plane" >}} de um cluster Kubernetes usando a API do Kubernetes.
+Ferramenta de linha de comando para se comunicar com o {{< glossary_tooltip text="camada de gerenciamento" term_id="control-plane" >}} de um cluster Kubernetes usando a API do Kubernetes.
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/kubectl.md:15]: Very minor: "o camada de gerenciamento" should be "a camada de gerenciamento".

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]

File: content/pt-br/docs/reference/glossary/kubectl.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: Kubectl
+id: kubectl
+date: 2018-04-12
+full_link: /docs/user-guide/kubectl-overview/
```
Comment (Sea-n) [inline content/pt-br/docs/reference/glossary/kubectl.md:5]:
```suggestion
full_link: /pt-br/docs/user-guide/kubectl-overview/
```

**Comment by Sea-n** [comment]
Thanks for quick fixing! But I am not a Brazilian Portuguese speaker, format-wise looks good to me. 😅
/uncc

**Comment by edsoncelio** [comment]
Thanks!
/lgtm 

@stormqueen1990 PTAL

**Review by stormqueen1990 — APPROVED** [review]

**Comment by rikatz** [comment]
/approve
Thanks

**Pull request merged**